odoo/api.py

```diff
@@ -687,7 +687,8 @@ def guess(method):
         return method
 
     # introspection on argument names to determine api style
-    args, vname, kwname, defaults = getargspec(method)
+    # Changed implementation for lazy support to Type Annotations
+    args, vname, kwname, defaults, kwonlyargs, kwonlydefaults, ann = getfullargspec(method)
```

Comment:
We handle the additional variables retrieved from **getfullargspec**, but only **args and kwname** are used